src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/ExecutableStatementCountCheck.java

```diff
@@ -29,7 +29,7 @@
 
 /**
  * <p>
- * Restricts the number of executable statements to a specified limit.
+ * Restricts the number of executable statements to a specified limit (default = 30).
```

Comment:
just to note that here is a default value

Reply:
done

Reply:
@pbludov , @strkkk , @rnveach ,

I think It is reasonable to remove default values from summary. We always have default for all properties. We do change defaults sometime, so we can miss update of summary, it will be hard to catch.

Reply:
I unresolved this item, as I need to know your thoughts on this.

Reply:
@romani This was already discussed and agreed upon at https://github.com/checkstyle/checkstyle/pull/7456#discussion_r365034934 . Is there something more you are looking for?

Reply:
No, resolved